src/handlers/flow/FlowStreamHandler.cpp

```diff
@@ -47,6 +67,44 @@ void FlowStreamHandler::start()
 
     process_groups(_group_defs);
 
+    // Setup Configs
+    if (config_exists("recorded_stream")) {
+        _metrics->set_recorded_stream();
+    }
+
+    if (config_exists("device_map")) {
+        for (const auto &device_info : config_get<StringList>("device_map")) {
+            std::vector<std::string> data = split(device_info, ',');
+            if (data.size() < 2) {
+                // should at least contain device name and ip
+                continue;
+            }
+            DeviceEnrich *device{nullptr};
+            if (auto it = enrich_data.find(data[1]); it != enrich_data.end()) {
+                device = &it->second;
+            } else {
+                enrich_data[data[1]] = DeviceEnrich{data[0], {}};
+                device = &enrich_data[data[1]];
+            }
+            if (data.size() < 4) {
+                // should have interface information
+                continue;
+            }
+            auto if_index = static_cast<uint32_t>(std::stol(data[3]));
+            if (auto it = device->interfaces.find(if_index); it == device->interfaces.end()) {
+                if (data.size() > 4) {
+                    device->interfaces[if_index] = InterfaceEnrich{data[2], data[4]};
+                } else {
+                    device->interfaces[if_index] = InterfaceEnrich{data[2], std::string()};
+                }
+            }
+        }
+    }
+
+    if (config_exists("concatenate_interface")) {
```

Comment:
@rboucher-ns1 the idea here is to pass like the data that you want to concatenate the `deviceId`
like 
```
config:
   concatenate_interface: "eth0"
```

Then, based on that we generate the label `interface="1.2.3.4.5|eth0` and if it has enrichment it would be `interface="route1|eth0`

Reply:
Would like your opinion oh that name `concatenate_interface`